test/bigtest/interactors/ConfirmationModalInteractor.js

```diff
@@ -0,0 +1,9 @@
+import {
+  clickable,
+  interactor,
+} from '@bigtest/interactor';
+
+export default interactor(class ConfirmationModalInteractor {
```

Comment:
it's a copy-paste, we must have similar in stripes-acq-comps

Reply:
You haven't used ConfirmationModalInteractor, is it OK?

Reply:
I used `ConfirmationInteractor` from `stripes-acq-components` in `unreceive-piece-test.js`